### PR TITLE
Added basic landing page

### LINKED ISSUE #2: Create a good landing page
Need to create a simple landing page explaining what this app is all about.

### LINKED ISSUE #31: [enhancement] make the *go to user's dashboard* button more specifying
### What would you like to share?

the current style doesn't feel good and users can be confused due to that, so it's better to make it look like a button instead

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by carrotfarmer**
## What does this PR fix?
- Potentially fixes #2

## Proposed Changes
- Changed site title to `Hashnode Stats` (previously `Create Next App`)
- Added a mono space font to the home page (if you like it, I will add it to the other pages too)
- Added some basic information about the site in the landing page including a description and the GitHub link

## Check List (Check all the applicable boxes) 
- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All tests have passed.

## Screenshots
image

## Additional Info
I've also added some hover effects which you can check out in the live deployment!

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -16,20 +16,45 @@ export default function Home() {
...
 
       <form className="flex flex-col space-y-2" onSubmit={handleSubmit}>
+        <h1 className="text-center text-gray-50 text-2xl font-mono font-extrabold">
+          Hashnode Stats
+        </h1>
+        <p className="text-center text-gray-50 text-md font-mono">
+          This website shows various statistics of{" "}
+          <a
+            href="https://hashnode.com"
+            target={"_blank"}
```
Comment (avneesh0612) [inline pages/index.tsx:31]: Where you are using target="_black" you should also use `rel="noreferrer"`

File: pages/index.tsx [hunk 2]
```diff
@@ -16,20 +16,45 @@ export default function Home() {
...
+          >
+            hashnode
+          </a>{" "}
+          users like bio information, posts etc.
+        </p>
+        <p className="text-center text-gray-50 text-md font-mono">
+          This project is open-source and feel free to star it on{" "}
+          <a
+            href="https://github.com/avneesh0612/hashnode-analytics"
+            target={"_blank"}
```
Comment (avneesh0612) [inline pages/index.tsx:42]: Same here, add- `rel="noreferrer"`

**Comment by carrotfarmer** [comment]
@avneesh0612 Made the changes. I also had doubts regarding what info to display in the landing page, so if there's anything missing, please let me know.

**Comment by avneesh0612** [comment]
Okay cool. Thanks for your contribution :)

**Pull request merged**